docs/resources/Guild.md

```diff
@@ -19,8 +19,8 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 | region                        | string                                                                              | [voice region](#DOCS_RESOURCES_VOICE/voice-region-object) id for the guild                                                       |
 | afk_channel_id                | ?snowflake                                                                          | id of afk channel                                                                                                                |
 | afk_timeout                   | integer                                                                             | afk timeout in seconds                                                                                                           |
-| embed_enabled?                | boolean                                                                             | whether this guild is embeddable (e.g. widget)                                                                                   |
-| embed_channel_id?             | ?snowflake                                                                          | if not null, the channel id that the widget will generate an invite to                                                           |
+| embed_enabled?                | boolean                                                                             | whether or not the server widget is enabled (deprecated, replaced with widget_enabled)                                           |
```

Comment:
I know it was server before as well. But shouldn't ``guild`` be preferred? Considering this is a guild object structure and not a server object structure. Maybe a nitpick. 
Otherwise looks ok.

Reply:
They're referred to as servers in many other parts of this page too

Reply:
I guess it should be fixed in a separate PR. It's just annoying with all these inconsistency